Composer/packages/client/src/recoilModel/dispatchers/utils/project.ts

```diff
@@ -377,8 +379,16 @@ export const fetchProjectDataById = async (projectId): Promise<{ botFiles: any;
   }
 };
 
-export const handleProjectFailure = (callbackHelpers: CallbackInterface, ex) => {
-  setError(callbackHelpers, ex);
+export const handleProjectFailure = (callbackHelpers: CallbackInterface, error) => {
+  const isDotnetError = checkIfDotnetVersionMissing({
+    message: error.response?.data?.message ?? error.message ?? '',
+  });
+
+  if (isDotnetError) {
+    callbackHelpers.set(warnAboutDotNetState, true);
+  } else {
+    setError(callbackHelpers, error);
```

Comment:
missing ```callbackHelpers.set(warnAboutDotNetState, fase);``` here?

Reply:
It would be unlikely that users would be able to get to _a different error_ before resolving the dotnet one.

But to ensure future code paths.. I'll add it!